### DISCUSSION TITLE
Input type="number"

### DISCUSSION THREAD

**Discussion by kivanchuk** [twilio-labs/paste#2356] [Q&A] [answered]
Hello! It seems, that Input type="number" is not supported yet. When is that possible to add this type support?

**Comment by TheSisb** [comment, marked as answer]
Hi @kivanchuk,

We [do support `input type="number"`](https://paste.twilio.design/components/input#number) --- [props table](https://paste.twilio.design/components/input#input-props). Are you having any specific issues with it? 

Under the hood we convert it to `text` based on [findings by the UK Gov design system](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/).
